### PR TITLE
Adding waiter to fix visibility of phoneNumber label

### PR DESCRIPTION
**Pull request by sherrif10**
This pr handles visibility of phone number label occured while fixing this ticket   https://github.com/openmrs/openmrs-contrib-qaframework/pull/54
Here are the logs that [returned the error to  be fixed](https://github.com/openmrs/openmrs-contrib-qaframework/pull/54/checks?check_run_id=3182664038)  cc @ibacher  @dkayiwa

### CODE REVIEW

**Comment by sherrif10** [comment]
cc @kdaud , Feel free to review  thanks

**Comment by sherrif10** [comment]
Hello @dkayiwa  @ibacher  @kdaud   still waiting as this is blocking me for the moment to proceed with RATEST-150

File: ui-tests/src/test/java/org/openmrs/reference/page/RegistrationPage.java [hunk 1]
```diff
@@ -166,6 +167,7 @@ public void clickOnContactInfo() throws InterruptedException {
     }
 
     public void clickOnPhoneNumber() throws InterruptedException {
+    	waiter.until(ExpectedConditions.visibilityOfElementLocated(PHONE_NUMBER_LABEL));
```
Comment (kdaud) [inline ui-tests/src/test/java/org/openmrs/reference/page/RegistrationPage.java:170]: Indention here! Correspond to the existing code indention.

**Comment by kdaud** [comment]
@sherrif10  One other thing: where a PR depends on another PR, it would be good to have a link in the PR to that other PR.

**Comment by sherrif10** [comment]
Did you see this https://github.com/openmrs/openmrs-contrib-qaframework/pull/54

File: ui-tests/src/test/java/org/openmrs/reference/page/RegistrationPage.java (hunk 1 above)
Comment (sherrif10) [reply ui-tests/src/test/java/org/openmrs/reference/page/RegistrationPage.java]: Oh sorry i forgot that, Fixed that

**Comment by sherrif10** [comment]
Resolved thnks

**Comment by sherrif10** [comment]
cc @dkayiwa  @ibacher

**Comment by dkayiwa** [comment]
@sherrif10 has @kdaud approved this pr?

Secondly, are all these changes needed? https://github.com/openmrs/openmrs-distro-referenceapplication/pull/515/files

**Comment by sherrif10** [comment]
@kdaud  btw is not used to approve pull request

**Pull request closed without merging**